### ISSUE TITLE
Dash: problem opening stream containing multiple $RepresentationID$ in URL

### ISSUE THREAD

**Issue by gizmocuz** [peak3d/inputstream.adaptive#893]
There are streams with multiple **$RepresentationID$** in the URL like:

https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd

Sollution is to change the following function

```
static void ReplacePlaceHolders(std::string& rep, const std::string& id, uint32_t bandwidth)
{
    while (1)
    {
        std::string::size_type repPos = rep.find("$RepresentationID$");
        if (repPos == std::string::npos)
            break;
        rep.replace(repPos, 18, id);
    }
    std::string::size_type repPos = rep.find("$Bandwidth$");
    if (repPos != std::string::npos)
    {
        char bw[32];
        sprintf(bw, "%u", bandwidth);
        rep.replace(repPos, 11, bw);
    }
}

```

**Comment by gizmocuz** [comment]
I created 3 PR's ... Not sure if this is the right way to do this when a project has 3 branches

**Comment by basilgello** [comment]
Right now, only master (Nexus) and Matrix are supported Kodi versiobs. Leia is no more supported, in case you made a PR targeting 18.x.

**Comment by glennguy** [comment]
Fixed in #898
